src/Nordea/Components/Button.elm

```diff
@@ -196,17 +211,45 @@ variantStyle variant =
                 ]
 
         Card ->
+            let
+                arrowIconMovement movementLengthRight =
+                    descendants
+                        [ everything
+                            [ withAttribute "type=arrowicon"
+                                [ transforms [ translateX movementLengthRight ]
+                                , transition [ Css.Transitions.transform3 150 0 easeOut ]
+                                ]
+                            ]
+                        ]
+            in
             batch
-                [ backgroundColor Colors.white
+                [ displayFlex
+                , backgroundColor Colors.white
                 , textAlign left
+                , alignItems start
                 , padding (rem 1)
                 , borderRadius (rem 0.75)
                 , borderStyle none
                 , boxShadow4 (rem 0) (rem 0.25) (rem 2.5) Colors.grayLight
-                , hover [ transform (scale 1.25) ]
+                , arrowIconMovement (rem 0)
+                , hover [ backgroundColor Colors.grayHover, arrowIconMovement (rem 1) ]
                 ]
 
 
 buttonStyleForExport : Variant -> Style
 buttonStyleForExport variant =
     batch [ baseStyle, variantStyle variant ]
+
+
+
+-- EXTRAS
+
+
+withStyles : List Style -> Button msg -> Button msg
+withStyles styles (Button config) =
+    Button { config | styles = styles }
+
+
+withHtmlTagWrapper : (List (Attribute msg) -> List (Html msg) -> Html msg) -> Button msg -> Button msg
+withHtmlTagWrapper htmlTag (Button config) =
```

Comment:
Just nitpicking 😛 As the tag isn't wrapping the component, but the tag of the component itself, I think we should avoid wrapper in the name

```suggestion
withHtmlTag : (List (Attribute msg) -> List (Html msg) -> Html msg) -> Button msg -> Button msg
withHtmlTag htmlTag (Button config) =
```